inabox/README.md

```diff
@@ -159,7 +159,7 @@ Disperse a blob:
 # To install `grpcurl`, run `brew install grpcurl` or `go install github.com/fullstorydev/grpcurl/cmd/grpcurl@latest`
 
 # From top level eigenda directory
-$ grpcurl -plaintext -d '{"data": "'$(tools/kzgpad/bin/kzgpad -e hello)'"}' \
+$ grpcurl -plaintext -d '{"data": "'$(kzgpad -e hello)'"}' \
```

Comment:
This means the dev has kzgpad already installed right?

Reply:
Can it be done through mise?

Reply:
added instructions [da2d626](https://github.com/Layr-Labs/eigenda/pull/1971/commits/da2d626bf97f03c7de24e4199ecdb8dd174dd171)